test/NuGet.Core.FuncTests/NuGet.Commands.FuncTest/RestoreCommandTests.cs

```diff
@@ -1347,6 +1348,94 @@ public async Task RestoreCommand_RestoreWithNoChangesAsync()
+                                    ""version"" : ""1.0.0"",
+                                }
+                            }
+                        }
+                    }
+                }");
+
+                var configuration = SimpleTestSettingsContext.GetOrAddSection(pathContext.Settings.XML, "config");
+                SimpleTestSettingsContext.AddEntry(configuration, "cacheExpiration", "true");
+                pathContext.Settings.Save();
+
+                // Arrange
+                var packageA = new SimpleTestPackageContext("a", "1.0.0");
+                packageA.Files.Clear();
+                packageA.AddFile("lib/net5.0/a.dll");
+
+                await SimpleTestPackageUtility.CreateFolderFeedV3Async(
+                    pathContext.PackageSource,
+                    PackageSaveMode.Defaultv3,
+                    packageA);
+
+                var sources = new List<PackageSource>
+                {
+                    new PackageSource(pathContext.PackageSource)
+                };
+                var logger = new TestLogger();
+                var settings = new Settings(pathContext.WorkingDirectory.Path);
+
+                var projectDirectory = Path.Combine(pathContext.SolutionRoot, "TestProject");
+                var cachingSourceProvider = new CachingSourceProvider(new PackageSourceProvider(NullSettings.Instance));
+
+                var spec = JsonPackageSpecReader.GetPackageSpec(configJson.ToString(), "TestProject", Path.Combine(projectDirectory, "project.csproj")).WithTestRestoreMetadata();
+                var dgSpec = new DependencyGraphSpec();
+                dgSpec.AddProject(spec);
+                dgSpec.AddRestore(spec.Name);
+
+                var request = new TestRestoreRequest(spec, sources, pathContext.UserPackagesFolder, ClientPolicyContext.GetClientPolicy(settings, logger), logger)
+                {
+                    ProjectStyle = ProjectStyle.PackageReference,
+                    DependencyGraphSpec = dgSpec,
+                    AllowNoOp = true,
+                    ForceUpdatePackageLastAccessTime = true
+                };
+                var command = new RestoreCommand(request);
+
+                // Preconditions
+                var result = await command.ExecuteAsync();
+                await result.CommitAsync(logger, CancellationToken.None);
+                result.Success.Should().BeTrue(because: logger.ShowMessages());
+                var newRequest = new TestRestoreRequest(spec, sources, pathContext.UserPackagesFolder, ClientPolicyContext.GetClientPolicy(settings, logger), logger)
+                {
+                    ProjectStyle = ProjectStyle.PackageReference,
+                    DependencyGraphSpec = dgSpec,
+                    AllowNoOp = true,
+                    ForceUpdatePackageLastAccessTime = true
+                };
+
+                var metadataPath = Path.Combine(pathContext.UserPackagesFolder, @"a\1.0.0\.nupkg.metadata");
```

Comment:
```suggestion
                var metadataPath = Path.Combine(pathContext.UserPackagesFolder, "a", "1.0.0", ".nupkg.metadata");
```

This is what caused the test to fail on Linux.

Reply:
Thanks